service/history/workflow/mutable_state.go

```diff
@@ -89,6 +89,14 @@ type (
 
 		// Indicate type of the current workflow task (normal, transient, or speculative).
 		Type enumsspb.WorkflowTaskType
+
+		// These two fields are sent to workers in the WorkflowTaskStarted event. We need to save a
+		// copy in mutable state to know the last values we sent (which might have been in a
+		// transient event), otherwise a dynamic config change of the suggestion threshold could
+		// cause the WorkflowTaskStarted event that the worker used to not match the event we saved
+		// in history.
```

Comment:
I guess this comment also needs to be updated.

Reply:
what part is wrong? I just updated this one